### DISCUSSION TITLE
`text-start` and `text-end` utilities

### DISCUSSION THREAD

**Discussion by lukewarlow** [tailwindlabs/tailwindcss#5788] [Ideas]
Tailwind should add a utility for `text-align: start` and `text-align: end` for better RTL support

**Comment by mnr73** [comment]
it's true, actually not only for RTL is useful in grids.  when in grid cols use text-left or text-right its not work until use direction too but some time we need in col some item placed at right or left without change direction
![2021-10-28_12-41-59](https://user-images.githubusercontent.com/23061025/139225837-36153165-af49-459b-9b03-1df52637a272.png)
![2021-10-28_12-42-57](https://user-images.githubusercontent.com/23061025/139225844-ae45f3cd-f11c-420e-9501-9dcb92e45d99.png)

it can be done with text-align: end;

can read more info here

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align

please add these classes
